### PR TITLE
Document the Go dashboard API

### PR DESCRIPTION
**Pull request by smn**
Specifically JSON & the HTTP API for publishing stats from 3rd party apps.

### CODE REVIEW

**Comment by smn** [comment]
@justinvdm & @hodgestar I know there are two more widgets available, the histogram and the pie chart. Where can I find example definitions for those?

Anyway, not particularly proud of these docs but it's better than what we have now.

**Comment by smn** [comment]
@imsickofmaps care to read over as well?

**Comment by justinvdm** [comment]
@smn don't think we have examples of those yet, will gist something.

**Comment by justinvdm** [comment]
@smn https://gist.github.com/justinvdm/82486f054c330c93eff2

**Comment by smn** [comment]
thanks, ready for review.

File: docs/dashboards.rst [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+Available Widgets
+~~~~~~~~~~~~~~~~~
+
+The following widgets are available:
+
+LValueWidget
+^^^^^^^^^^^^
+
+Displays the last value of a given metric together with the last value of
+the same metric for the given time range in history. Makes it easy to
```
Comment (hodgestar) [inline docs/dashboards.rst]: I found the first sentence here rather confusing. It uses `last value` twice to mean different things and I'm not sure what `the given time range in history` means.

File: docs/http_api.rst [hunk 2]
```diff
@@ -1,9 +1,18 @@
+.. _http_api:
+
 Vumi Go's HTTP API
 ==================
 
-The API allows for sending & receiving Vumi messages via HTTP. These messages are plain JSON strings. Two types of messages are available, TransportUserMessages and TransportEvents. These messages can be received via streaming HTTP or can be pushed to 3rd party URL via HTTP POST. Outbound messages & metrics can be pushed to Vumi Go via HTTP PUT.
+The API allows for sending & receiving Vumi messages via HTTP. These
+messages are plain JSON strings. Two types of messages are available,
+TransportUserMessages and TransportEvents. These messages can be received
+via streaming HTTP or can be pushed to 3rd party URL via HTTP POST.
```
Comment (hodgestar) [inline docs/http_api.rst:9]: Are we officially deprecating the streaming HTTP API?

**Comment by hodgestar** [comment]
Otherwise looks good.

File: docs/dashboards.rst (hunk 1 above)
Comment (smn) [inline docs/dashboards.rst]: It is super confusing and my english is failing me. Can you suggest a better way to describe the LValueWidget?

Comment (hodgestar) [inline docs/dashboards.rst]: @justinvdm Could you maybe suggestion something here? The implementation seems to return values at `now() - time_range` and `now() - 2 * time_range` with `now()` possibly floored to the last whole interval of `time_range`, which seems sort of odd and complicated?

Comment (smn) [inline docs/dashboards.rst]: @justinvdm if you've got time to update this that'd be great, then I can ping Daniela to let the guys needing know that the docs have been published.

**Pull request merged**